src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -32,6 +33,7 @@ internal class MgmtOutputLibrary : OutputLibrary
         private Dictionary<string, List<OperationGroup>> _operationGroups;
         private Dictionary<string, TypeProvider> _nameToTypeProvider;
         private IEnumerable<Schema> _allSchemas;
+        private Dictionary<Operation, LongRunningOperation>? _longRunningOperations;
```

Comment:
At some point you should consider moving to the CachedDictionary the rest of the code is using. I don't see any side effects here to be worried of.